### ISSUE TITLE
Looking for some help 

### ISSUE THREAD

**Issue by Aech40** [LNSterne/LD14P_RP2040#1]
Hi, 
I am trying to implement this Lidar on my own for an autonomous RC car. I am trying to run your example code and every time I run  it i get loads of errors. Do you have any insight?

Thanks!

![image](https://github.com/LNSterne/LD14P_RP2040/assets/104017490/7c9275bc-6b23-498c-95b9-cd5b8ad3a80f)

**Comment by LNSterne** [comment]
Hi,

What board and core are you using?

The error is ocurring becuause the compiler can't find the C++ standard libraries. I have found with the standard Arduino boards that these are not provided and have to download a seperate library for these.

The board and core I have tested this on is the standard Raspberry Pi Pico using this core https://github.com/earlephilhower/arduino-pico

**Comment by Aech40** [comment]
Yep I added that core and it uploaded. I started with a regular Uno R3, then went and got a pico to replicate your setup. Additional question, What does your example sketch actually do? Should i see output or something or is it just the driver to make it work? I am reading it right where it says to connect the lidar on pins 4 and 5, yes? 

Thank you for all your help!

**Comment by LNSterne** [comment]
The example should print out to serial the angle and distance values around 180 degrees. 

Yes the lidar should be connected to GPIO 4 and 5, which is physical pins 6 and 7.
